App/Reducers/LoginReducer.js

```diff
@@ -9,20 +9,24 @@ export const INITIAL_STATE = Immutable({
 })
 
 // login attempts
-const attempt = (state, action) =>
-  state.merge({ attempting: true })
+const attempt = (state, action) => {
+  return { ...state, ...{ attempting: true } }
```

Comment:
this would be written
`return { ...state, attempting: true}`

but that doesn't return an Immutable object (requirement of reducers by convention).

So it would become
`return Immutable({ ...state, attempting: true})`

Now, @skellock  is a fan of `merge` over spread operator.  Can you go into more detail on why this change was needed so we can make sure we're taking the best path.